mypy/newsemanal/semanal.py

```diff
@@ -4544,8 +4553,11 @@ def schedule_patch(self, priority: int, patch: Callable[[], None]) -> None:
         self.patches.append((priority, patch))
 
     def report_hang(self) -> None:
+        print('Deferral trace:')
+        for mod, line in self.deferral_debug_context:
+            print('    {}: {}'.format(mod, line))
```

Comment:
Maybe leave out the space after `:`, for consistency with mypy error message formatting.